elements/pf-alert/pf-alert.css

```
@@ -1,65 +1,65 @@
 :host {
-  --pf-alert-title-color: var(--pf-alert-title-color, var(--pf-global--default-color--300, #003737));
-  --pf-alert-icon-color: var(--pf-alert-icon-color, var(--pf-global--default-color--200, #009596));
-  --pf-alert-inline-background-color: var(--pf-global--palette--cyan-50, #f2f9f9);
+  --alert-title-color: var(--alert-title-color, var(--pf-global--default-color--300, #003737));
```

```suggestion
  --_alert-title-color: var(--pf-global--default-color--300, #003737);
```
etc, and at call sites